tests/common.py

```diff
@@ -1646,3 +1647,50 @@ def extract_stack_to_frame(extract_stack: list[Mock]) -> FrameType:
         current_frame = next_frame
 
     return top_frame
+
+
+_EntityT = TypeVar("_EntityT", bound=MockEntity | MockToggleEntity)
```

Comment:
What's the point of this typevar, it's not used?

Reply:
mypy (although we don't use it for tests 🙂) complained when I used `entities: list[MockEntity | MockToggleEntity],` in the function definition, but I'm totally fine to use `entities: list[MockEntity | MockToggleEntity],` insteam

Reply:
Ok, nvm, using `Sequence` solved this.